dev/release/post-12-bump-versions.sh

```diff
@@ -64,7 +64,7 @@ if [ ${BUMP_VERSION_POST_TAG} -gt 0 ]; then
 fi
 
 if [ ${BUMP_DEB_PACKAGE_NAMES} -gt 0 ] && \
-     [ "${next_version}" != "$(current_version)" ]; then
+     [ "${next_version}" != "${version}" ]; then
```

Comment:
Hmm.
`$(current_version)` is a function call not a variable reference.
So `current_version` function call is intentional here.